roles/pulp_common/tasks/install_pip.yml

```diff
@@ -132,6 +132,11 @@
         virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
       when: prereq_pip_packages | length > 0
 
+    - name: Install a patched version of Django LTS (FIPS only)
+      pip:
+        name: git+https://github.com/mdellweg/django.git@fips
+      when: ansible_fips
```

Comment:
So if we want to manually test this even without FIPS enabled on the system, we would just override `ansible_fips` using higher precedence variable methods (like the group_vars, or on the command line.)

Reply:
I think, you can even inject facts for certain hosts.

https://docs.ansible.com/ansible/devel/user_guide/playbooks_vars_facts.html#adding-custom-facts

But can we also get container with those assertions for md5?

Reply:
So I ran a test. Users can override ansible_fips, but not ansible_facts.fips. And ansible_facts.fips is required to meet certain user's requirements:
https://github.com/pulp/pulp_installer/pull/432

I think we should add a boolean variable `pulp_fips` or `pulp_fips_mode`. The default value will be that of `ansible_facts.fips`